Changelog.md

```diff
@@ -1,8 +1,39 @@
-## SDFormat 9.X
+## libsdformat 9.X
 
-### SDFormat 9.X.X (202X-XX-XX)
+### libsdformat 9.X.X (202X-XX-XX)
 
-### SDFormat 9.3.0 (2020-XX-XX)
+### libsdformat 9.4.0 (2020-12-XX)
+
+1. Define `PATH_MAX` for Debian Hurd system
+    * [Pull request 369](https://github.com/osrf/sdformat/pull/369)
+
+1. Fix supported shader types (`normal_map_X_space`)
+    * [Pull request 383](https://github.com/osrf/sdformat/pull/383)
+
+1. Move list of debian dependencies to packages.apt
+    * [Pull request 392](https://github.com/osrf/sdformat/pull/392)
+
+1. Enable/disable tests for issue #202, add macOS workflow
+    * [Pull request 414](https://github.com/osrf/sdformat/pull/414)
+    * [Pull request 438](https://github.com/osrf/sdformat/pull/438)
+    * [Issue 202](https://github.com/osrf/sdformat/issues/202)
+
+1. Test included model folder missing model.config
+    * [Pull request 422](https://github.com/osrf/sdformat/pull/422)
+
+1. Prefix nested model names when flattening
+    * [Pull request 399](https://github.com/osrf/sdformat/pull/399)
+
+1. Add Sky DOM
+    * [Pull request 417](https://github.com/osrf/sdformat/pull/417)
+
+1. camera.sdf: decrease far clip lower bound
+    * [Pull request 435](https://github.com/osrf/sdformat/pull/435)
+
+1. material.sdf: add `double_sided` parameter
+    * [Pull request 435](https://github.com/osrf/sdformat/pull/435)
```

Comment:
Is this #410?

Reply:
nice catch; fixed in 5c211e50c709cd073c4b2a25fd9f05f64b5587da